apps/core/templates/components/header.html

```diff
@@ -10,9 +10,26 @@
           <span class="fw-bold me-2">wagtail</span>
           <span>editor's guide</span>
         </a>
-        <a href="{% url 'search' %}" class="bg-transparent border-0 border-start border-1 border-dark ps-4 ms-4">
+        <button class="bg-transparent border-0 border-start border-1 border-dark ps-4 ms-4" data-bs-toggle="modal" data-bs-target="#searchModal">
           <img src="{% manifest 'images/search.svg' %}" >
-        </a>
+        </button>
+        <div class="modal fade" id="searchModal" tabindex="-1" aria-labelledby="searchModalLabel" aria-hidden="true">
+          <div class="modal-dialog modal-lg">
+            <div class="modal-content rounded-3 fs-6">
+              <div class="modal-body p-4 d-flex align-items-center">
+                <input type="text" placeholder="Search..." class="border border-1 border-dark rounded-start w-75 p-2 ps-3" name="search" />
```

Comment:
Can we keep this a form for now? So it opens the search page on submit?

We can do the live search results in anther PR.